tid_project/src/components/layout/ProfileBar.js

```diff
@@ -34,9 +34,9 @@ const ProfileBar = () => {
   useEffect(() => {
     const getCurrentUser = async () => {
       try {
-        const currentUser = Parse.User.current();
-        if (currentUser) {
-          setUsername(currentUser.get('username'));
+        const response = await apiClient.get('/users/me');
+        if (response.data) {
+          setUsername(response.data.username);
         } else {
           setUsername('Guest');
```

Comment:
What made you decide to change how you implemented Parse?

Reply:
I was not sure if we were to refactor our use of Parse when we implemented REST api service?